Packs/CommonDashboards/Scripts/UseCaseAdoptionMetrics/UseCaseAdoptionMetrics.py

```diff
@@ -0,0 +1,117 @@
+        return False
+    except DemistoException as e:
+        return DemistoException(str(e))
+
+
+def get_use_cases() -> Dict[str, Any]:
+    """
+    Retrieves use cases data based on modules in Demisto.
+
+    Returns:
+        dict: A dictionary containing use cases in production and at risk.
+    """
+    use_cases_in_production = set()
+    at_risk = []
+
+    phishing_incidents = check_phishing_incidents()
+    use_case_dict = {
+        'Ransomware & Malware Coverage': 'https://cortex.marketplace.pan.dev/marketplace/?useCase=Malware',
+        'Business Email Compromise Coverage': 'https://cortex.marketplace.pan.dev/marketplace/?useCase=Phishing',
+        'Network Security': 'https://xsoar.pan.dev/marketplace/?category=Network%20Security',
+        'Analytics & SIEM': 'https://cortex.marketplace.pan.dev/marketplace/?category=Analytics+%26+SIEM',
+        'Data Enrichment & Threat Intelligence': 'https://cortex.marketplace.pan.dev/marketplace/?category=Data+Enrichment+%26+Threat+Intelligence',
+        'Vulnerability Management': 'https://xsoar.pan.dev/marketplace/?category=Vulnerability%20Management',
+        'Case Management': 'https://cortex.marketplace.pan.dev/marketplace/?category=Case+Management',
+        'Rapid Breach Response': 'https://cortex.marketplace.pan.dev/marketplace/details/MajorBreachesInvestigationandResponse/'
+    }
+
+    for _, details in demisto.getModules().items():
+        category = details.get('category', '').lower()
+        brand = details.get('brand', '').lower()
+        state = details.get('state')
+        incident_types = details.get('incident_types', [])
+
+        if brand != 'builtin' and state == 'active' and category != 'utilities':
+            if category in ['forensic & malware analysis', 'endpoint']:
+                use_cases_in_production.add('Ransomware & Malware Coverage')
+            elif category in ['email', 'messaging', 'messaging and conferencing'] and 'phishing' in incident_types:
+                if phishing_incidents:
+                    use_cases_in_production.add('Business Email Compromise Coverage')
+                else:
+                    at_risk.append(
+                        '[Business Email Compromise Coverage](https://xsoar.pan.dev/marketplace/?category=Email%2C%20Messaging)')
+            elif category == 'network security':
+                use_cases_in_production.add('Network Security')
+            elif category == 'analytics & siem':
+                use_cases_in_production.add('Analytics & SIEM')
+            elif category == 'data enrichment & threat intelligence':
+                use_cases_in_production.add('Data Enrichment & Threat Intelligence')
+            elif category == 'vulnerability management':
+                use_cases_in_production.add('Vulnerability Management')
+            elif category == 'case management':
+                use_cases_in_production.add('Case Management')
```

Comment:
Lets make this a dict and if the category is in it we will take the mapped name and add it to use_cases_in_production.